src/sentry/apidocs/examples/replay_examples.py

```diff
@@ -50,6 +50,20 @@ class ReplayExamples:
         )
     ]
 
+    GET_REPLAY_CLIKS = [
+        OpenApiExample(
+            "Retrieve a collection of RRWeb DOM node-ids and the timestamp they were clicked.",
+            value={
+                "data": [
```

Comment:
I was also wondering, is there a reason why we have this `data` key instead of returning the list itself?

Reply:
Because there may be properties other than the data itself that we want to return some day.  Links, count of items.